Integrate-Kobiton.md

```diff
@@ -125,35 +107,32 @@ There are two ways to set your username and API key, you can choose either one o
 **Setting Username and API key as global variables**
 
 1. Go to [CircleCI dashboard](https://circleci.com/dashboard)
-2. Click `Settings` button in the left panel of the page.
 
-![alt text](./assets/settings-button-left-panel.png)
-
-3. In `Settings` window, under `Organization` section, click `Projects`
+2. In the left panel of the page, click `Settings` -> `Projects`
 
 ![alt text](./assets/projects-button.png)
 
-4. In `Followed projects` section, click the gear icon in the project you want to configure.
+3. In `Followed projects` section, click the gear icon in the project you want to configure.
 
 ![alt text](./assets/configure-project-icon.png)
 
-5. In the left pane of the window, click `Environment variables` button under `Build Settings` section
+4. In the left pane of the window, click `Environment variables` button under `Build Settings` section
 
 ![alt text](./assets/env-variables-button.png)
 
-6. In `Environment Variables` window, click `Add Variable`
+5. In `Environment Variables` window, click `Add Variable`
 
 ![alt text](./assets/add-variable-button.png)
 
-7. In `Add an Environment Variable` popup, fill in `Name` field with the one in the picture (marked **red**). In `Value` field, replace `YOUR_KOBITON_USERNAME` (marked **blue**) with your Kobiton's username
+6. In `Add an Environment Variable` popup, fill in `Name` field with the one in the picture (marked **red**). In `Value` field, replace `YOUR_KOBITON_USERNAME` (marked **blue**) with your Kobiton's username
 
 ![alt text](./assets/kobi-username-env.png)
 
-8. Click `Add Variable` button
+7. Click `Add Variable` button
 
 ![alt text](./assets/add-env-variable-button.png)
 
-Repeat step 6, 7, 8, replace `YOUR_KOBITON_API_KEY` (marked **blue**) with your Kobiton's API key 
+Repeat step 5, 6, 7, replace `YOUR_KOBITON_API_KEY` (marked **blue**) with your Kobiton's API key 
 
 ![alt text](./assets/kobi-api-env.png)
 
```

Comment:
Aren't we supposed to modify our testing script to access theses environment variables?